fix faro.receiver cors not allowing x-scope-orgid, traceparent headers (#4051)
grafana/alloy: docs/sources/reference/components/faro/faro.receiver.md

To support requests from all origins, set `cors_allowed_origins` to `["*"]`.
The `*` character indicates a wildcard.

> The allowed headers for cross-domain requests are hardcoded as: `Content-Type`, `Traceparent`, `X-API-Key`, `X-Faro-Session-Id`, `X-Scope-OrgID`.

Reviewer: ```suggestion
You can use the following headers for cross-domain requests: `Content-Type`, `Traceparent`, `X-API-Key`, `X-Faro-Session-Id`, `X-Scope-OrgID`.
```

I don't think we need to state they are hardcoded... just say "use these headers".

Author: This sounds better, thank you! 👍
